src/serialize.cpp

```diff
@@ -1,6 +1,7 @@
 #include <migraphx/serialize.hpp>
 #include <migraphx/argument.hpp>
 #include <migraphx/literal.hpp>
+#include <nlohmann/json.hpp>
```

Comment:
This should be added to a json.cpp file, not in the same serialize header.

Reply:
fixed